apps/codeforafrica/src/components/GuidingPrinciplesCard/GuidingPrinciplesCard.js

```diff
@@ -0,0 +1,36 @@
+import { RichTypography } from "@commons-ui/core";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import React from "react";
+
+const GuidingPrinciplesCard = React.forwardRef(function GuidingPrinciplesCard(
+  props,
+  ref
+) {
+  const { title, description, icon } = props;
+  return (
+    <Card
+      sx={{
+        borderRadius: 0,
+        boxShadow: "none",
+        marginTop: { lg: "40px", xs: "20px" },
```

Comment:
This is not part of the Card... should be in the component responsible for laying out cards.